src/ensembl/src/content/app/browser/track-panel/trackPanelActions.ts

```diff
@@ -82,6 +81,26 @@ export const changeTrackPanelModalViewForGenome: ActionCreator<
   );
 };
 
+export const setHighlightedTrack: ActionCreator<
```

Comment:
How about `changeHighlightedTrack` instead?

Reply:
That's done